tests/test_wellstate.cpp

```diff
@@ -440,6 +440,10 @@ BOOST_AUTO_TEST_CASE(TESTWellContainer) {
     BOOST_CHECK_EQUAL(wc.well_name(1), "W2");
     BOOST_CHECK_THROW(wc.well_name(10), std::exception);
 
+    const auto& wells = wc.wells();
+    BOOST_CHECK_EQUAL(wells.size() , wc.size());
+    BOOST_CHECK( (wells[0] == "W1" && wells[1] == "W2") ||
+                 (wells[1] == "W1" && wells[0] == "W2") );
```

Comment:
It's probably overkill for two-element sequences,  but in general you can use [`std::is_permutation()`](https://en.cppreference.com/w/cpp/algorithm/is_permutation)&mdash;preferably in the dual-range form&mdash;to answer these kinds of questions.

Reply:
Well overkill or not - `std::is_permutation()` was a good tool to learn about.